api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/UpdatableClaimantField.java

```diff
@@ -0,0 +1,125 @@
+package uk.gov.dhsc.htbhf.claimant.model;
+
+import lombok.Getter;
+import uk.gov.dhsc.htbhf.claimant.entity.Address;
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+
+import java.util.Objects;
+
+/**
+ * Enumeration of all the fields on a Claimant that may be updated by the claimant.
+ */
+@Getter
+public enum UpdatableClaimantField {
```

Comment:
Yes, this is much better and clearer than the first attempt using reflection, I like this! :-)